cluster/gce/util.sh

```diff
@@ -433,6 +433,13 @@ function kube-up {
     --target-tags "${MASTER_TAG}" \
     --allow tcp:443 &
 
+  # We have to make sure the disk is created before creating the master VM, so
+  # run this in the foreground.
+  gcloud compute disks create "${MASTER_NAME}-pd" \
+    --project "${PROJECT}" \
+    --zone "${ZONE}" \
+    --size "10GB"
+
```

Comment:
Can you explain the re-organizations in here, both of the `compute disks create` and the `MINION_SCOPES`? They don't seem to be necessary?

If it's just cleanup, maybe revert or put in a separate commit?